ADR/0013-integration-service-api-contracts.md

```diff
@@ -36,20 +36,22 @@ Related to [PLNSRVCE-41](https://issues.redhat.com/browse/PLNSRVCE-41) investiga
 
 The output of each Tekton task will be provided in a minimized [Tekton result](https://tekton.dev/docs/pipelines/tasks/#emitting-results) in JSON format listing all test failures. The name of the result will be **HACBS_TEST_OUTPUT**.
 
+To display count of found vulnerabilities and make it easy to understand and evaluate the state of scanned image, the additional output of Tekton task `clair-scan` will be provided in a minimized [Tekton result](https://tekton.dev/docs/pipelines/tasks/#emitting-results) in JSON format listing. The name of the result will be **CLAIR_SCAN_RESULT**.
+
 The maximum size of a [Task's Results](https://tekton.dev/vault/pipelines-v0.17.3/tasks/#emitting-results) is limited by the container [termination message](https://kubernetes.io/docs/tasks/debug/debug-application/determine-reason-pod-failure/#customizing-the-termination-message) feature of Kubernetes.
 
 App Studio builds are structured as [a shared Persistent Volume per AppStudio Workspace](https://docs.google.com/document/d/1IPlihVjkJ4Kb9tdhsk7iz3bn5rkT_SvJCNQhyXzK3aI/edit#bookmark=id.gefgys3vno2). This allows teams to share builds, implement caching and other shared volumes. A single persistent volume is mapped to each default build pipeline. Builds are passed a directory specific to their builds.
 
-#### Tekton Result Format
+#### Tekton Result Format for `HACBS_TEST_OUTPUT`
 
-The Test output of the Tekton result will be a JSON object that includes the context about the test along with the list of check names for all failed checks
+The Test output of the Tekton result **HACBS_TEST_OUTPUT** will be a JSON object that includes the context about the test along with the list of check names for all failed checks
 
 The output will provide the following information about the overall test result:
 - **result** - The outcome of the testing task, can be `SUCCESS`, `FAILURE`, `WARNING`, `SKIPPED` or `ERROR`
-- **namespace** - Optional, the rego namespace of the test policy
+- **namespace** - The default value is `default`. It is set to the rego namespace for the test policy
```

Comment:
Nit: Is there another way we can say this?

> The default value is `default`.

Not critical to get it changed as I don't have a better wording off the top of my head. It just feels like it could be better.

Reply:
How about changing to 'The rego namespace for the test policy or the fault value `default`'?

Reply:
It reads fine as it is, I was just trying to get rid of the double-default. Another option could be something like
> The rego namespace for the test policy. If not set, it will be assumed to have the value `default`.

Reply:
Yes, looks better! Thanks. Let me update it.

Reply:
Updated.